Utilities/ocpasswordgen/ocpasswordgen.c

```diff
@@ -21,13 +58,34 @@ ENTRY_POINT (
   UINT8   Index;
   UINT8   PasswordHash[SHA512_DIGEST_SIZE];
 
-  DEBUG ((DEBUG_ERROR, "Please enter your password: "));
+  DEBUG ((DEBUG_ERROR, "Please enter your password: \n"));
 
   for (PasswordLen = 0; PasswordLen < OC_PASSWORD_MAX_LEN; ++PasswordLen) {
-    Char = getchar ();
-    if ((Char == EOF) || (Char == '\n')) {
+    fflush(stdin);
+    Char = getch ();
+    // Handle interrupt signal for conio
+    if (Char == CHAR_END_OF_TEXT) {
+      exit(EXIT_SUCCESS);
```

Comment:
Why is that success? The password was not generated, in my opinion, that's `EXIT_FAILURE`.

Reply:
Yep, it should be EXIT_FAILURE